sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/Transforms.java

```diff
@@ -244,39 +244,57 @@ private Transforms() {
         FormField<?> value;
         switch (fieldValue.getType()) {
             case PHONE_NUMBER:
-                value = new FormField<>(setDefaultConfidenceValue(fieldValue.getConfidence()), labelText,
-                    key, fieldValue.getValuePhoneNumber(), valueText, FieldValueType.PHONE_NUMBER);
+                value = new FormField<>(key, labelText, valueText, new com.azure.ai.formrecognizer.models.FieldValue(FieldValueType.PHONE_NUMBER).
+                setFormFieldPhoneNumber(fieldValue.getValuePhoneNumber()), setDefaultConfidenceValue(fieldValue.getConfidence())
+                );
                 break;
             case STRING:
-                value = new FormField<>(setDefaultConfidenceValue(fieldValue.getConfidence()), labelText,
-                    key, fieldValue.getValueString(), valueText, FieldValueType.STRING);
+                value = new FormField<>(key, labelText, valueText, new com.azure.ai.formrecognizer.models.FieldValue(FieldValueType.STRING)
```

Comment:
nit: com.azure.ai.formrecognizer.models.FieldValue can be in import list at the beginning of file

Reply:
Can't..have FieldValue in implementation.models too.